include/nuttx/net/ip.h

```diff
@@ -238,6 +238,51 @@ extern "C"
  * Public Function Prototypes
  ****************************************************************************/
 
+/****************************************************************************
+ * Name: net_iphdrsize
+ *
+ * Description:
+ *   Return the size of the IP header
+ *
+ * Input Parameters:
+ *   domain  - IP domain: PF_INET or PF_INET6
+ *   l4size  - L4 additional header size
+ *
+ * Returned Value:
+ *   The size of the combined L3 + L4 headers is returned.
+ *
+ ****************************************************************************/
+
+static inline uint16_t net_iphdrsize(uint8_t domain, uint16_t l4size)
```

Comment:
We are introducing inline to common code again :(

Reply:
emm.... OK i will move to c code

Reply:
Done